Core/EventHandlers.lua

```diff
@@ -1361,6 +1361,35 @@ function R:OnEvent(event, ...)
 			end
 		end
 
+		-- Handle opening lots of various chests for Gilded Wader (pet).
+		if
+			Rarity.isFishing and Rarity.isOpening and Rarity.lastNode and
+				(Rarity.lastNode == L["Gift of Thenios"] or Rarity.lastNode == L["Hidden Hoard"]
+				or Rarity.lastNode == L["Memorial Offerings"] or Rarity.lastNode == L["Treasure of Courage"])
+		 then
+			local names = {"Gilded Wader"}
+			if (Rarity.lastNode == L["Gift of Thenios"]) then
```

Comment:
Since you have already checked if `Rarity.lastNode` is equal to one of the localized chest names in the outer `if` a few lines above, can't you just do the following and get rid of the if/else conditional altogether?

``` lua
Rarity:Debug("Detected Opening on " .. Rarity.lastNode .. " (method = SPECIAL)")
```

Reply:
Yeah. It's just brainless search/replace copy/pasta. I'll clean it up after the rest is done 👌